index.html

```diff
@@ -148,44 +154,86 @@ <h2 class="salutation">Tonic</h2>
+
+        <p class="connect">LET'S CONNECT</p>
+
+        <section class="social">
+            <ul >
+                <li><a href="https://twitter.com/home"><img src="images/twitter.jpg" alt="twitter"></a></li>
+                <li> <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin-icon.jpg" alt="Linkedin"></a></li>
+                <li><a href="https://twitter.com/home"><img src="images/Vector.jpg" alt="twitter"></a></li>
+                <li><a href="https://github.com/ausiemass/portfolio"><img src="images/github.jpg" alt="GITHUB"></a></li>
+                <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
+            </ul> 
+        </section>   
+        
+        <button class="get-resume" type="button">Get my Resume</button>     
 
     </div>
+
     
-    <section class="skills">
-
-        <ul class="about-skill">
-            <li class="aboutl">Languages <img src="images/arrowDown.jpg" alt=""></li> 
-            <p><img src="images/js.jpg" alt="">JavaScript</p>
-            <p><img src="images/html.jpg" alt="">HTML</p>
-            <p><img src="images/ccs3.jpg" alt="">CSS</p>
-            
+        <div class="skill-list">
+            <ul>
+                <li>
+                    <span class="skills">
+                        Languages <span><img class="skills-img1" src="./images/arrowDown.jpg" alt="languages"></span>
+                    </span>
+                    <ul class="skills-languages">
+                        <li class="add-skills">
+                            <a href="#"><img src="images/js.jpg" alt="JS"></a>
+                            <span class="in-text">
+                                JavaSript
+                            </span>
+                        </li>
+                        <li class="add-skills">
+                            <a href="#"><img  src="images/html.jpg" alt="HTML"></a>
+                            <span class="in-text">
+                                HTML
+                            </span>
+                        </li>
+                        <li class="add-skills">
+                            <a href="#"><img  src="images/ccs3.jpg" alt="CSS"></a>
+                            <span class="in-text">
+                                CSS
+                            </span>
+                        </li>
+                    </ul> 
+                </li>
+                <li class="skills">Frameworks <span><img class="skills-img2" src="images/arrowRight.jpg" alt="frameworks"></span></li>
+                 <li>
+                    <hr class="line">
+                 </li>
+                <li class="skills">Skills <span><img class="skills-img2" src="images/arrowRight.jpg" alt="Skills"></span></li>
+                <li>
+                    <hr class="line">
+                </li>
+            </ul>
+        </div>
+
```

Comment:
- [ ] [Optiional] On the right side of the project `language, framework and skills` your list is not matching the color on the design please fix this issue. you can compare the design below
### Your design
![his](https://github.com/ausiemass/portfolio-austin/assets/81727988/251539c7-ae22-4b7e-b3ef-f34a44a3ae98)

### Figma design
![fimga](https://github.com/ausiemass/portfolio-austin/assets/81727988/c58becdc-5e44-4c63-bfe4-911af85758c4)